### PR TITLE
Add contributing page

### PR DESCRIPTION
**Pull request by swantzter**

### CODE REVIEW

File: config/footer.yml [hunk 1]
```diff
@@ -41,6 +41,8 @@
       link: 'kb/index/all'
     - name: Supporting the project
       link: pages/support
+    - name: Contributing
```
Comment (swantzter) [inline config/footer.yml:44]: boo hoo the footer columns won't be even anymore

File: content/pages/contributing.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+base.
+
+[philosophy]: https://freenode.net/philosophy
+[github]: https://github.com/freenode
+[roadmap]: https://github.com/orgs/freenode/projects/1
+
+## The Development Team
+
+When contributing to freenode you will get to interact closely with our
+development team which comprises of, amongst others:
```
Comment (ilbelkyr) [inline content/pages/contributing.md:58]: "comprises" doesn't take "of" afaik; it also feels a little weird to use it with "amongst others", but that may just be me?

**Review by ilbelkyr — CHANGES_REQUESTED** [review]

File: content/pages/contributing.md (hunk 2 above)
Comment (swantzter) [reply content/pages/contributing.md:58]: yeah that's possible, I don't really English :p Any suggestion you think works better?

Comment (jesopo) [reply content/pages/contributing.md:58]: I think you can just strip the "of"

Comment (swantzter) [reply content/pages/contributing.md]: done

**Pull request merged**